definitions/variable/emissions/tag_ocean-removal-option.yaml

```diff
@@ -0,0 +1,7 @@
+- Ocean Removal Option:
+    - Ocean Alkalinity Enhancement:
+        description: increasing ocean alkalinity
+    - Direct Ocean Capture:
+        description: direct ocean capture
```

Comment:
"direct removal of CO2 from the ocean through electrodialysis or other methods"

Reply:
Done